src/results/heating_cooling_load.jl

```diff
@@ -94,3 +94,17 @@ function add_heating_load_results(m::JuMP.AbstractModel, p::REoptInputs, d::Dict
     d["HeatingLoad"] =   r
     nothing
 end
+
+"""
+MPC `HeatingLoad` results keys:
+- `process_heat_thermal_load_series_mmbtu_per_hour` vector of site space heating boiler load in every time step
```

Comment:
I'm slightly confused about why it's only process heat some places but others there's also dhw and space heating